### PR TITLE
Update README to include tunneling information

### PR DESCRIPTION
**Pull request by alissapajer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The configuration of the Kafka Datasource has the following JSON format:
 
 ```
 {
- "bootstrapServers": Array of Strings "host:port",
+ "bootstrapServers": Array of Strings,
```
Comment (dcsobral) [inline README.md:15]: Was this change necessary? Unlike other places, Kafka _requires_ the `:port` part of it.

File: README.md [hunk 2]
```diff
@@ -34,15 +34,21 @@ The configuration of the Kafka Datasource has the following JSON format:
     "escape": String
   }
   [, "compressionScheme": "gzip"]
+  [, "tunnelConfig": {
+    host: String,
+    port: Int,
+    pass: <PASS>
```
Comment (dcsobral) [inline README.md:40]: It's `auth`, not `pass`.

File: README.md [hunk 3]
```diff
@@ -34,15 +34,21 @@ The configuration of the Kafka Datasource has the following JSON format:
...
 ```
 
-*
-*
-*
++ `bootstrapServers` is an Array of Strings of the form "host:port"
++ `groupId` is the Kafka group id
++ `topics` are the Kafka topics to connect to
++ `decoder` indicates if the key (`"RawKey"`) or value (`"RawValue"`) of the Kafka object should be decoded
++ `pass` is of the form `{ "password": String }` | `{ "key": String, "passphrase": String }` where `pass.key` is content of private key file for ssh tunneling
```
Comment (dcsobral) [inline README.md:49]:
```
s/pass\b/auth/g
```

File: README.md [hunk 4]
```diff
@@ -54,7 +60,12 @@ Example
    "type": "json",
    "variant": "line-delimited",
    "precise": false
- }
+ },
+ "tunnelConfig": {
+    host: "host_name",
+    port: 22222,
+    pass: { "password": "my_secret_password" }
```
Comment (dcsobral) [inline README.md:67]: s/pass/auth/

**Review by dcsobral — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alissapajer) [reply README.md:15]: I'll change it back

**Comment by alissapajer** [comment]
@dcsobral ready for another look

**Comment by dcsobral** [comment]
I just realized now it's been sitting in the queue for 8 days. My github notifier stopped working at some point, and I didn't get a notification on my phone either. :(

**Review by dcsobral — APPROVED** [review]
